sdk/python/kubeflow/training/constants/constants.py

```diff
@@ -128,6 +130,9 @@
         "plural": PYTORCHJOB_PLURAL,
         "container": PYTORCHJOB_CONTAINER,
         "base_image": PYTORCHJOB_BASE_IMAGE,
+        "init_container": STORAGE_CONTAINER,
+        "init_container_image": STORAGE_CONTAINER_IMAGE,
+        "train_container_image": TRAINER_TRANSFORMER_IMAGE,
```

Comment:
I think, you can remove it from this dict, since these consts will be the same for all Job types.